servicetalk-grpc-netty/src/main/java/io/servicetalk/grpc/netty/DefaultGrpcClientBuilder.java

```diff
@@ -237,4 +256,25 @@ public void doAppendHttpClientFilter(final Predicate<StreamingHttpRequest> predi
                                          final StreamingHttpClientFilterFactory factory) {
         httpClientBuilder.appendClientFilter(predicate, factory);
     }
+
+    private void appendCatchAllFilter() {
```

Comment:
Hm, I didn't know that we have "CatchAll" filter for the client. Not sure why it's necessary because `DefaultGrpcClientCallFactory` will map all exceptions correctly. Any thoughts on why it can be useful?